packages/cli/src/generator/printRunInstructions.js

```diff
@@ -22,17 +23,18 @@ function printRunInstructions(projectDir, projectName) {
     process.cwd(),
     xcodeProjectPath
   );
-  logger.info(`To run your app on iOS:
-    cd ${absoluteProjectDir}
-    react-native run-ios');
+
+  logger.log(`
+  ${chalk.cyan(`Run instructions for ${chalk.bold('iOS')}`)}:
+    • cd ${absoluteProjectDir} && react-native run-ios
```

Comment:
I'd prefer relative path, wdyt?

Reply:
Agreed. In general, absolute paths are prettier, but not in this case because we will always have `./project-dir`.